### PR TITLE
Add translation for zh-cn

### PR DESCRIPTION
**Pull request by Chaoba**
Add the latest translation for zh-cn

### CODE REVIEW

**Comment by hzsweers** [comment]
Thanks! Do you have someone that can read chinese and review this per our `CONTRIBUTING.md` guidelines?

**Comment by Chaoba** [comment]
My colleague @makulu1987 will review this PR.

**Comment by hzsweers** [comment]
Thanks!

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -270,5 +284,33 @@
     <string name="preview">预览</string>
     <string name="show_raw_markdown">显示 markdown 源码</string>
     <string name="render_markdown">渲染 markdown</string>
+    <string name="copy_hash">复制哈希</string>
+    <string name="toast_msg_copied">复制到剪贴板</string>
```
Comment (makulu1987) [inline app/src/main/res/values-zh-rCN/strings.xml]: “已复制” 或者 “已复制到剪贴板” 更合适一点

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -270,5 +284,33 @@
...
+
+    <!-- Deleting repo -->
+    <string name="are_you_sure">真的确定要删除吗？</string>
+    <string name="unexpected_bad_things">删除后将无法恢复。 版本库、wiki、issue和评论都将会被永久性的删除，所有的协作关系也会被移除。</string>
```
Comment (makulu1987) [inline app/src/main/res/values-zh-rCN/strings.xml]: repository、wiki、issue、以及comments都将被永久删除，并且无法恢复。同时，也会移除所有的协作关系。

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 3]
```diff
@@ -270,5 +284,33 @@
...
+    <string name="delete_successful">版本库已删除</string>
+    <string name="login_or_email">账号或电子邮箱</string>
+
+    <string name="temporary_login_message"><b>现在通过网站进行登录验证, 点击登录按钮继续。</b>
```
Comment (makulu1987) [inline app/src/main/res/values-zh-rCN/strings.xml]: 现在的授权过程是通过网页方式进行的，点击登录按钮以继续。

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 4]
```diff
@@ -270,5 +284,33 @@
...
+        \n\n 因为GitHub添加了对第三方应用的限制，我们无法写入特定的组织。你可以从这里获取更多信息：\n
```
Comment (makulu1987) [inline app/src/main/res/values-zh-rCN/strings.xml]: 由于Github添加了对第三方应用的限制，所以我们不能写成明确的组织。

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 5]
```diff
@@ -270,5 +284,33 @@
...
+        https://help.github.com/articles/about-third-party-application-restrictions/</string>
+    <string name="organizations">组织</string>
+    <string name="error_oauth_not_configured">你无法登陆，因为还没有设置认证客户端和秘钥
+阅读README.md: https://github.com/pockethub/PocketHub#setup-environment</string>
+
+
+    <string-array name="welcome_texts">
+        <item>不久之前GitHub app不再维护了，但是我们想要一些新的东西…</item>
+        <item>我们给app改了一个新的名字，现在叫做PocketHub</item>
+        <item>点击done我们会让你登录！</item>
```
Comment (makulu1987) [inline app/src/main/res/values-zh-rCN/strings.xml]: 点击done完成登录

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 6]
```diff
@@ -270,5 +284,33 @@
...
+    <string name="temporary_login_message"><b>现在通过网站进行登录验证, 点击登录按钮继续。</b>
+        \n\n 因为GitHub添加了对第三方应用的限制，我们无法写入特定的组织。你可以从这里获取更多信息：\n
+        https://help.github.com/articles/about-third-party-application-restrictions/</string>
+    <string name="organizations">组织</string>
+    <string name="error_oauth_not_configured">你无法登陆，因为还没有设置认证客户端和秘钥
+阅读README.md: https://github.com/pockethub/PocketHub#setup-environment</string>
+
+
+    <string-array name="welcome_texts">
+        <item>不久之前GitHub app不再维护了，但是我们想要一些新的东西…</item>
```
Comment (makulu1987) [inline app/src/main/res/values-zh-rCN/strings.xml]: 不久之前Github app停止更新了，但是我们中的一些人想在项目中添加一些新的东西...

**Comment by Chaoba** [comment]
Thanks @makulu1987 for his comments. We have discussed for a better translation.

**Comment by Harlber** [comment]
Perhaps ‘**你**’ replaced with '**您**' is more appropriate.
Btw, what does '**复制哈希**' mean in line 287 of `values-zh-rCN/strings.xml`

**Comment by Chaoba** [comment]
@ Herlber How to translate "Copy hash" in your opinion?

**Pull request merged**